xonsh/completers/bash.py

```diff
@@ -1,97 +1,42 @@
-import os
-import re
 import shlex
-import pickle
-import hashlib
 import pathlib
 import builtins
 import subprocess
 
-import xonsh.lazyasd as xl
 import xonsh.platform as xp
 
 from xonsh.completers.path import _quote_paths
 
-RE_DASHF = xl.LazyObject(lambda: re.compile(r'-F\s+(\w+)'),
-                         globals(), 'RE_DASHF')
-
-INITED = False
-
-BASH_COMPLETE_HASH = None
-BASH_COMPLETE_FUNCS = {}
-BASH_COMPLETE_FILES = {}
-
-CACHED_HASH = None
-CACHED_FUNCS = None
-CACHED_FILES = None
-
-BASH_COMPLETE_SCRIPT = """source "{filename}"
+BASH_COMPLETE_SCRIPT = r'''
```

Comment:
PEP8: triple double quotes instead of triple single quotes https://www.python.org/dev/peps/pep-0008/#string-quotes